Products/LDAPUserFolder/LDAPDelegate.py

```diff
@@ -400,7 +398,7 @@ def insert(self, base, rdn, attrs=None):
             else:
                 is_binary = False
 
-            if isinstance(attr_val, (str, unicode)) and not is_binary:
+            if isinstance(attr_val, (str, bytes)) and not is_binary:
```

Comment:
Won't the following line fail if the value is bytes because the `split` is passing a `str` value?

Reply:
Definitely, looking into it.